include/old/test.h

```diff
@@ -44,6 +44,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
+#ifndef __GLIBC__
+#include <bits/reg.h>
+#endif
 
```

Comment:
It's used in ~20 testcases as well, so we may end up defining TST_WORDSIZE or something based on __WORDSIZE (if defined and do some other check otherwise) and make use of that one instead.